activerecord/lib/active_record/relation/calculations.rb

```diff
@@ -36,9 +36,15 @@ module Calculations
     # Note: not all valid +select+ expressions are valid +count+ expressions. The specifics differ
     # between databases. In invalid cases, an error from the database is thrown.
     def count(column_name = nil, options = {})
+      if options.present? && !ActiveRecord.const_defined?(:DeprecatedFinders)
+        raise ArgumentError, "Relation#count does not support finder options anymore. " \
+                             "Please build a scope and then call count on it or use the " \
+                             "activerecord-deprecated_finders gem to enable this functionality."
```

Comment:
Ah, right it is! So used to things being against master I didn't think twice about it. Sorry about that, and thanks for working on this :grin: